src/MoBi.Presentation/DTO/QuantityDTO.cs

```diff
@@ -17,5 +17,8 @@ public IDimension Dimension
          get => Value.Dimension;
          set => Value.Dimension = value;
       }
+
+      public SimulationEntitySourceReference SourceReference { get; set; }
+      public string SourceDisplayName => SourceReference?.BuildingBlock?.DisplayName ?? string.Empty;
```

Comment:
in Container DTO you are ok with returning null but not in QuantityDTO (string.empty). why?